shell/components/Questions/Reference.vue

```diff
@@ -90,15 +117,17 @@ export default {
     class="row"
   >
     <div class="col span-6">
-      <LabeledSelect
-        :mode="mode"
-        :options="options"
+      <ResourceLabeledSelect
+        :resourceType="typeName"
+        :inStore="inStore"
```

Comment:
Interesting, I thought our linter prevented camelcase in our templates.

Reply:
I wasn't sure if this was now supported or not, no warnings so at dev time left as is.

I've updated them now though to be correct, in case the linting is fixed in future